namespaces/non-production.k8s.integration.dsd.io/formbuilder-services-dev/formbuilder-services-dev-admin-role.yaml

```diff
@@ -58,6 +58,9 @@ subjects:
   - kind: Group
     name: "github:form-builder"
     apiGroup: rbac.authorization.k8s.io
+  - kind: ServiceAccount
+    name: "formbuilder-platform-dev:formbuilder-publisher-dev"
```

Comment:
I believe this syntax will not work. Where is the `ServiceAccount` defined?

Reply:
I've now added a separate file in each namespace to create the service accounts